cellprofiler/modules/trackobjects.py

```diff
@@ -57,8 +57,15 @@
 `Examples <http://cellprofiler.org/examples/#Tracking>`__
 webpage.
 
-Available measurements
-^^^^^^^^^^^^^^^^^^^^^^
+*Note on saving images:* You can pass the modified objects along to the
```

Comment:
Maybe "identified objects"  or "tracked objects", instead of "modified objects"?